src/modules/claim/services/revocation-verification.service.ts

```diff
@@ -15,10 +15,10 @@ export class RevocationVerificationService extends RevocationVerification {
     didService: DIDService,
     roleService: RoleService,
     provider: Provider,
-    @Inject('RegistrySettings') registrySettings: RegistrySettings
+    @Inject('RegistrySettings') registrySettings: RegistrySettings,
+    revokerResolver: RoleRevokerResolver
   ) {
     const issuerResolver = new RoleIssuerResolver(roleService);
```

Comment:
Hopefully this worked 🤞 . But looking at it now, it seems a bit odd to only inject `RoleRevokerResolver` and not `RoleIssuerResolver`. What are your thoughts on this @whitneypurdum @JGiter ?